### PR TITLE
chore(core): mitigation cve dvcr-importer, vm-route-forge and bundle

### PR DESCRIPTION
**Pull request by universal-itengineer**
## Description

Vulnerabilities in the following images have been fixed: 
- dvcr-importer
Updated branch in 3p-cdi.
- bundle
Applied Python updates for hooks.
- vm-route-forge
Updated cilium for vm-route-forge.

Bundle image added to the scan process.

## Why do we need it, and what problem does it solve?

Vulnerability mitigation:
CVE-2024-28180
CVE-2024-47825
CVE-2024-39689
CVE-2024-26130
GHSA-h4gh-qq45-vh27
## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [X] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

**Review by yaroslavborbat — DISMISSED** [review]

**Review by Isteb4k — CHANGES_REQUESTED** [review]

File: images/dvcr-artifact/pkg/monitoring/progress.go [hunk 1]
```diff
@@ -183,50 +183,66 @@ func (p *ProgressMeter) updateSpeed() bool {
...
-	if err != nil {
-		return err
+type ProgressMetric interface {
+	Add(value float64)
+	Set(value float64)
+	Get() (float64, error)
+	Delete()
+}
+
+func NewProgress(importProgress *prometheus.GaugeVec, lvs ...string) *ImportProgress {
```
[the remark this answers is not in the document]
Comment (yaroslavborbat) [reply images/dvcr-artifact/pkg/monitoring/progress.go:215]: label values. Its a default naming in prometheus

File: images/dvcr-artifact/pkg/importer/importer.go [hunk 2]
```diff
@@ -177,22 +177,24 @@ func (i *Importer) runForDataSource(ctx context.Context) error {
 		return monitoring.WriteImportFailureMessage(err)
 	}
 
-	return monitoring.WriteImportCompleteMessage(res.SourceImageSize, res.VirtualSize, res.AvgSpeed, res.Format, durCollector.Collect())
+	return monitoring.WriteImportCompleteMessage(res.SourceImageSize, res.VirtualSize, res.AvgSpeed, res.Format,
+		durCollector.Collect())
```
[the remark this answers is not in the document]
Comment (yaroslavborbat) [reply images/dvcr-artifact/pkg/importer/importer.go:181]: maybe the linter did it

File: images/dvcr-artifact/pkg/monitoring/progress.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (yaroslavborbat) [reply images/dvcr-artifact/pkg/monitoring/progress.go]: renamed

**Review by yaroslavborbat — APPROVED** [review]

**Review by Isteb4k — DISMISSED** [review]

File: images/dvcr-artifact/go.mod [hunk 3]
```diff
@@ -3,126 +3,133 @@ module github.com/deckhouse/virtualization-controller/dvcr-importers
...
+	github.com/prometheus/client_golang v1.19.0
+	github.com/prometheus/client_model v0.6.0
+	golang.org/x/net v0.26.0 // indirect
+	golang.org/x/sync v0.7.0
+	k8s.io/klog/v2 v2.120.1
 	kubevirt.io/containerized-data-importer v0.0.0-00010101000000-000000000000
-	kubevirt.io/containerized-data-importer-api v1.57.0-alpha1
+	// kubevirt.io/containerized-data-importer-api v1.60.3
+	// kubevirt.io/containerized-data-importer-api v1.58.0-rc1
+	// kubevirt.io/containerized-data-importer-api v1.57.0-alpha1
```
Comment (universal-itengineer) [inline images/dvcr-artifact/go.mod:23]:
```suggestion
```

**Review by diafour — DISMISSED** [review]

**Review by Isteb4k — APPROVED** [review]

**Review by diafour — APPROVED** [review]

**Pull request merged**